nginx-autoinstall.sh

```diff
@@ -103,6 +103,10 @@ case $OPTION in
 		done
 		while [[ $GEOIP != "y" && $GEOIP != "n" ]]; do
 			read -rp "       GeoIP [y/n]: " -e GEOIP
+
+			while [[ $GEOIP = "y" && ! -n "$GEOIP_LICENSE_KEY" ]]; do
```

Comment:
Maybe add a statement like "If no key is provided, databases will not be downloaded automatically, you will need to download them yourself and put them in /opt/geoip, etc..."